src/content/getting-started/quickstart/external/_index.md

```diff
@@ -0,0 +1,429 @@
+      app: external-dns-cluster-a
+  template:
+    metadata:
+      labels:
+        app: external-dns-cluster-a
+    spec:
+      containers:
+      - name: dnsmasq
+        image: registry.access.redhat.com/ubi8/ubi-minimal:latest
+        ports:
+        - containerPort: 53
+        command: [ "/bin/sh", "-c", "microdnf install -y dnsmasq; ln -s /upstreamservers /etc/dnsmasq.d/upstreamservers; dnsmasq -k" ]
+        volumeMounts:
+        - name: upstreamservers
+          mountPath: /upstreamservers
+      volumes:
+        - name: upstreamservers
+          configMap:
+            name: external-dnsmasq
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: submariner-operator
+  name: external-dns-cluster-a
+spec:
+  ports:
+  - name: udp
+    port: 53
+    protocol: UDP
+    targetPort: 53
+  selector:
+    app: external-dns-cluster-a
+```
+
+Use this yaml to create DNS server, and assign global ingress IP:
+
+```bash
+kubectl apply -f dns.yaml
+subctl export service -n submariner-operator external-dns-cluster-a
+```
+
+Check global ingress IP:
+
+```bash
+kubectl --kubeconfig kubeconfig.cluster-a get globalingressip external-dns-cluster-a -n submariner-operator
+NAME                     IP
+external-dns-cluster-a   242.0.255.251
+```
+
+### Set up non-cluster hosts
+
+Modify routing for global CIDR on test-vm:
+
+Note that `subm_gw_ip` is the gateway node IP of the cluster in the same network segment of the hosts.
+In the case of the example of this guide, it is the node IP of cluster-a.
+
+```bash
+subm_gw_ip=192.168.122.26
+ip r add 242.0.0.0/8 via ${subm_gw_ip}
+```
+
+To persist above configuration across reboot, check the document for each Linux distribution.
+For example, on Centos 7, to set presistent route for eth0, below command is required:
+```bash
+echo "242.0.0.0/8 via ${subm_gw_ip} dev eth0" >> /etc/sysconfig/network-scripts/route-eth0
+```
+
+Modify `/etc/resolv.conf` to change DNS server for the host on test-vm:
+
+For example)
+
+- Before:
+```
+nameserver 192.168.122.1
+```
+
+- After:
+```
+nameserver 242.0.255.251
+```
+
+Check that the DNS server itself can be resolved:
+
+```bash
+nslookup external-dns-cluster-a.submariner-operator.svc.clusterset.local
+Server:         242.0.255.251
+Address:        242.0.255.251#53
+
+Name:   external-dns-cluster-a.submariner-operator.svc.clusterset.local
+Address: 10.43.162.46
+```
+
+### Verify Deployment
+
+#### Verify Manually
+
+##### Deploy HTTP server on hosts
+
+Run on test-vm:
+
+- python 2.x case:
+```bash
+python -m SimpleHTTPServer 80
+```
+- python 3.x case:
+```bash
+python -m http.server 80
+```
+
+##### Verify access to hosts from clusters
+
+Create Service, Endpoints, ServiceExport to access to test-vm, on cluster-a:
+
+Note that `Endpoints.subsets.addresses` needs to be modified to IP of test-vm.
+
+```bash
+export KUBECONFIG=kubeconfig.cluster-a
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Service
+metadata:
+  name: test-vm
+spec:
+  ports:
```

Comment:
Does it make sense to make it headless service, by adding `clusterIP: None`?
Then, it will be headless service without selector.

By doing so, I expect that it can provide a persistent source IP from the endpoint that is pointing to an external host.
Currently, it doesn't seem to work maybe the similar reason to this [issue](https://github.com/submariner-io/lighthouse/issues/603).

Reply:
Opened a separate issue https://github.com/submariner-io/submariner/issues/1537 .